dkms.in

```diff
@@ -426,6 +426,12 @@ safe_source() {
     )
     . "$tmpfile"
     rm "$tmpfile"
+
+    (( ${#REMAKE_INITRD[@]} )) && deprecated "REMAKE_INITRD ($to_source_file)"
+    (( ${#MODULES_CONF[@]} )) && deprecated "MODULES_CONF ($to_source_file)"
+    (( ${#MODULES_CONF_OBSOLETES[@]} )) && deprecated "MODULES_CONF_OBSOLETES ($to_source_file)"
+    (( ${#MODULES_CONF_ALIAS_TYPE[@]} )) && deprecated "MODULES_CONF_ALIAS_TYPE ($to_source_file)"
+    (( ${#MODULES_CONF_OBSOLETE_ONLY[@]} )) && deprecated "MODULES_CONF_OBSOLETE_ONLY ($to_source_file)"
```

Comment:
Why is this moved here? As-is it will cause the deprecation warnings to be printed more times, not less.

Reply:
postponed to a later pull request